support-frontend/assets/components/button/anchorButton.tsx

```diff
@@ -1,16 +1,19 @@
 // ----- Imports ----- //
+import type { EmotionJSX } from '@emotion/react/types/jsx-namespace';
 import type { SharedButtonPropTypes } from './_sharedButton';
 import SharedButton, { defaultProps } from './_sharedButton';
 import './button.scss';
+
 // ----- Render ----- //
 export type PropTypes = SharedButtonPropTypes & {
 	'aria-label'?: string | null | undefined;
 	href: string;
+	onClick?: () => void;
 };
 
-const AnchorButton = (props: PropTypes) => (
-	<SharedButton element="a" {...props} />
-);
+function AnchorButton(props: PropTypes): EmotionJSX.Element {
+	return <SharedButton element="a" {...props} />;
+}
```

Comment:
I think we've been standardising on `JSX.Element` as the return value from function components. It's compatible with the `EmotionJSX.Element` type so it should be straightforward to swap it out with `JSX.Element`

Reply:
Ah yep, I was blindly following what VSC suggested. Thanks Tom. Done!